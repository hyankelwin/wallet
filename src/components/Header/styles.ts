import styled from 'styled-components';
import Color from '../../styles/color';

export const Content = styled.div`
  width: 100%;
  height: 90px;
  display: flex;
  justify-content: space-between;
  align-items: center;
  background-color: ${Color.white};
  border-bottom: 1px solid #efefef;
  margin: 0 auto;

  @media (max-width: 400px) {
    width: 107%;
  }

  @media (max-width: 320px) {
    width: 113%;
  }

  div {
    display: flex;

    &:nth-child(1) {
      margin-left: 20px;
    }

    p {
      margin: 0;
      &:nth-child(1) {
        padding-right: 5px;
      }

      &:nth-child(2) {
        padding-right: 10px;
        color: ${Color.primary.default};
      }

      @media (max-width: 420px) {
        display: none;
      }
    }
  }
`;
