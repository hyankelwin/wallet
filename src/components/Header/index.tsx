import React from 'react';
import { Content } from './styles';
import HeaderImage from '../HeaderImage';

const Header: React.FC = () => {
  return (
    <Content>
      <div>
        <HeaderImage />
      </div>
      <div>
        <p>Your wallet</p>

        <p>digital</p>
      </div>
    </Content>
  );
};

export default Header;
