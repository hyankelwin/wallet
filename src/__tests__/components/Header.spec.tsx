import { mount } from 'enzyme';
import Header from '../../components/Header';

describe('Header component', () => {
  it('should be able to render Header component with success', () => {
    const wrapper = mount(<Header />);

    expect(wrapper.debug()).toMatchSnapshot();
  });
});
