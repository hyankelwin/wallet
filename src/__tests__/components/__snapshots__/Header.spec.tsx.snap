// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should be able to render Header component with success 1`] = `
"<Header>
  <styled.div>
    <div className=\\"sc-bdvvtL gfWDWR\\">
      <div>
        <HeaderImage>
          <styled.div>
            <div className=\\"sc-gsDKAQ gwkgJf\\" />
          </styled.div>
        </HeaderImage>
      </div>
      <div>
        <p>
          Your wallet
        </p>
        <p>
          digital
        </p>
      </div>
    </div>
  </styled.div>
</Header>"
`;
